daily-challenges/2024-10/21/khanh5923_21.spec.ts

```
@@ -0,0 +1,36 @@
+import { test, expect } from '@playwright/test';
+test.describe('khanh5923_20-Buy ticket', () => {
+    function convertPrice(price: string) {
+        return parseInt(price.replace(/\D/g, ''));
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)